src/service/middlewares/article-exists.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const {HttpCode} = require(`../../constants`);
+
+module.exports = (service) => async (req, res, next) => {
+  const {articleId} = req.params;
+  const article = await service.findOne(articleId);
+
+  if (!article) {
+    return res.status(HttpCode.NOT_FOUND)
+      .send(`Article with ${articleId} not found`);
+  }
+
+  return next();
```

Comment:
для консистентности с другими модулями
```suggestion
  next();
```